spec/requests/apartments_spec.rb

```diff
@@ -1,7 +1,36 @@
 require 'rails_helper'
 
 RSpec.describe "Apartments", type: :request do
-  describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
+  describe "POST /create" do
+    it"creates a new listing" do
+      apartment_params = {
+        apartment: {
+          street: "random way",
+          city: "random City",
+          state: "Tennessee",
+          manager: "random manager",
+          email: "email@email.com", 
+          price: 2200, 
+          bedrooms: 2, 
+          bathrooms: 2, 
+          pets: "not allowed",
+          image: "https://images.unsplash.com/photo-1605283176568-9b41fde3672e?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=774&q=80",
+          user_id:1
```

Comment:
Users ids should ideally not be hard coded. It would be a good idea to make a user instance and create an apartment from that user. Check out the README in the project for an example.